package-scripts/omnibus-toolchain/postrm

```diff
@@ -8,6 +8,22 @@
 # - if you are under 40, get peer review from your elders.
 #
 
+cleanup_symlinks() {
+  binaries="git ruby tar gtar bash berks bundle curl fakeroot gem make gmake makedepend patch pkg-config rake"
+  for binary in $binaries; do
+    rm -f $PREFIX/bin/$binary
+  done
+}
+
+# Clean up binary symlinks if they exist
+if [ ! -f /etc/redhat-release -a ! -f /etc/fedora-release -a ! -f /etc/system-release -a ! -f /etc/SuSE-release ]; then
+  # not a redhat-ish RPM-based system
+  cleanup_symlinks
+elif [ "x$1" = "x0" ]; then
```

Comment:
Do we need `=` for strict POSIX compliance here?  It's not a huge deal, but if we don't, `==` is more readable to those of us who have it ingrained to look out for `=` inside `==` inside conditional expressions.